docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
 * [Tessera](../HowTo/GetStarted/Install.md#as-release-binaries).
 * [IBFT network as configured in IBFT tutorial](Create-IBFT-Network.md).
 
-### Steps
-
-1. Generate new keys using `java -jar /path-to-tessera/tessera.jar -keygen -filename new-node-1`
-
-    ```bash
-    mkdir new-node-1t
-    cd new-node-1t
-    java -jar ../tessera.jar -keygen -filename new-node-1
-    ```
-
-    Result:
-
-    ```text
-    Enter a password if you want to lock the private key or leave blank
-    ```
-
-    Enter a password.
-
-    ```text
-    Please re-enter the password (or lack of) to confirm
-    ```
-
-    Enter it again.
-
-    Result:
-
-    ```text
-    10:32:51.256 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generating new keypair...
-    10:32:51.279 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generated public key PublicKey[pnesVeDgs805ZPbnulzC5wokDzpdN7CeYKVUBXup/W4=] and private key REDACTED
-    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved public key to /Users/username/fromscratch/new-node-1t/new-node-1.pub
-    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved private key to /Users/username/fromscratch/new-node-1t/new-node-1.key
-    ```
-
-1. Create a `new-node-1t/config.json` configuration file with newly generated keys referenced:
-
-    ```json
-    {
-       "useWhiteList": false,
-       "jdbc": {
-           "username": "sa",
-           "password": "",
-           "url": "jdbc:h2:/yourpath/new-node-1t/db1;MODE=Oracle;TRACE_LEVEL_SYSTEM_OUT=0",
-           "autoCreateTables": true
+## Steps
+
+Listed on the right-hand side of the page are the steps to create a private network using IBFT
+with 5 GoQuorum nodes and 2 Tessera nodes.
+
+### 1. Create directories
+
+Create directories for the 2 Tessera nodes in the `IBFT-Network` directory previously created.
```

Comment:
Rework the tree output to display both Tessera 0 and 1 one under the other, there's no reason for them to be appart as they should follow each other in alphabetical order.

```
IBFT-Network/
├── Node-0
│   └── data
├── Node-1
│   └── data
├── Node-2
│   └── data
├── Node-3
│   └── data
├── Node-4
│   └── data
├── Tessera-0
└── Tessera-1
```

Reply:
Fixed